index.html

```diff
@@ -7,6 +7,7 @@
     <meta name="theme-color" content="#2A3A4B">
     <!--#include virtual="base.html" -->
 
+    <link rel="icon" href="images/favicon.svg">
```

Comment:
this should go on title.html, also please update the Makefile to drop the reference to the old file.

Reply:
Ok, I'll work on this and also test for Safari.

Reply:
@saghul 

About testing on Safari, I have Windows so I cannot test on Safari.

This [page](https://support.apple.com/en-us/HT204416) confirms that _Apple no longer offers Safari updates for Windows. Safari 5.1.7 for Windows was the last version made for Windows, and it is now outdated._

Also, Apple's website only offers the latest version of Safari, so I can't even get 5.1.7.

Reply:
@saghul I made the other changes that you suggested.